### ISSUE TITLE
webpack odd behavior (nothing literally happens - probably stalls)

### ISSUE THREAD

**Issue by shiroganee** [jtangelder/sass-loader#213]
Well I've been reading issues over here and I believe it is the same case as mentioned in few places here and there. that's similar to #100 
Anyway how does it look in my project.

Whenever I try to import anything scss related - it stops working - 
![image](https://cloud.githubusercontent.com/assets/9503398/12726119/1dc805ee-c916-11e5-8817-a48597ab32cf.png)

If I will add it manually during HRM then it works. (First I start application without import, later on I add that import to file and it works (and show what I uploaded below))
![image](https://cloud.githubusercontent.com/assets/9503398/12726057/d4519cd6-c915-11e5-92bd-ce5aa42064e7.png)

I'm quite a newbie to Webpack though so maybe I'm doing something absolutely wrong.

and 4 commands that i'm using currently

```
"clean": "rimraf dist",
    "build:webpack": "webpack --config webpack.config.prod.js  --progress --colors",
    "build": "npm run clean && npm run build:webpack",
    "start": "npm run clean && npm run build:webpack && node server.js",

```

**Comment by shiroganee** [comment]
Ok I solved it by myself. The problem wasn't with the sass-loader nor with the webpack (sorry). It was all about 
`typeof window !== 'undefined.`
I'm writing isomorphic app and the webpack nor the sass-loader was crashing. The crashing one was application during start. Odd thing that WebStorm didn't show that. I only observed that in Visual Studio Code (unexpected token in scss file).

`if (typeof window !== 'undefined') `
and require in that resolved problem and it is working like a charm right now.
